javascript/javascript2/week3/app.js

```diff
@@ -0,0 +1,126 @@
+const button = document.querySelector("#button");
+const buttonLogLocation = document.querySelector("#log-location");
+const divCoordinates = document.querySelector("#coordinates");
+document.addEventListener("DOMContentLoaded", function () {
+  setTimeout(logAfterTwoSeconds, 2500);
+});
```

Comment:
In the exercise it is supposed to be when the script is loaded. So you don't need the Domcontentloaded but it's fine to have it!